### PR TITLE
Revert "Use ``importlib`` instead of ``pkg_resources`` for determining namespace packages"

### PR DESCRIPTION
**Pull request by DanielNoord**
Reverts PyCQA/astroid#1326

See discussion in https://github.com/PyCQA/astroid/pull/1406.

Sad 😢

### CODE REVIEW

File: astroid/const.py [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import enum
 import sys
```
Comment (DanielNoord) [inline astroid/const.py:3]:
```suggestion

PY36 = sys.version_info[:2] == (3, 6)
```

**Review by Pierre-Sassoulas — APPROVED** [review]
Getting rid of ``pkg_resources``won't be easy, but a lot of preliminary work was already done and the situation is still better than in 2.9, thank you @DanielNoord 👍

**Comment by DanielNoord** [comment]
> Getting rid of `pkg_resources`won't be easy, but a lot of preliminary work was already done and the situation is still better than in 2.9, thank you @DanielNoord 👍

With some debug tips from @cdce8p I'm certain we can figure this out. I know much more about `importlib` now so we're probably on the right track already 😄

**Review by cdce8p — APPROVED** [review]
LGTM. I left some debug tips [here](https://github.com/PyCQA/astroid/pull/1406#issuecomment-1053537537).

**Pull request merged**